action-server/tests/actions/test_daily_ci_enroll_form.py

```diff
@@ -619,7 +632,8 @@ def test_provide_preconditions_explain(self):
         with self.assertRaises(ActionExecutionRejection):
             self.run_form(tracker)
 
-    def test_provide_has_dialogue_affirm(self):
+    @patch("covidflow.actions.daily_ci_enroll_form.ci_enroll", return_value=True)
```

Comment:
The real thing doesn't return anything now, the mock should return None.

Reply:
Indeed, leftover from before